hw03_finagnon.py

```diff
@@ -0,0 +1,299 @@
+
+#################################################################################
+# Author:Finagnon Adjagbodjou
+# Username: Fin
+#
+# Assignment: hw03
+# Purpose: Learn the purpose of pulling, pushing, etc.
+# Google Doc Link: https://docs.google.com/document/d/1BjZMQNg6d2PG-PlwE5gFo9XBrmMei0aY6B_CtXdtFLg/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements: 
+#
+#
+#################################################################################
+
+import turtle
+import turtle
+import random
+
+def rightturn(shape):
```

Comment:
Pay attention to the gray squigglies, they indicate style guidelines. This one says there should be two blank lines before every function definition.